lochness/rpms/__init__.py

```diff
@@ -120,32 +121,33 @@ def initialize_metadata(Lochness: 'Lochness object',
 
 
 def get_subject_data(all_df_dict: Dict[str, pd.DataFrame],
-                     subject: object) -> Dict[str, pd.DataFrame]:
+                     subject: object,
+                     id_colname: str) -> Dict[str, pd.DataFrame]:
     '''Get subject data from the pandas dataframes in the dictionary'''
 
     subject_df_dict = {}
     for measure, measure_df in all_df_dict.items():
-        subject_df = measure_df[measure_df.record_id1 == subject.id]
+        measure_df[id_colname] = measure_df[id_colname].astype(str)
```

Comment:
Was ` measure_df[id_colname]` being interpreted as `int` /`float` in some cases before?

Reply:
in the CSV files used in this `proof of concept` flow test, the unique IDs were set as integers. I may remove this later when Prescient network provides us the table with more realistic IDs, which are in string.